helix-view/src/document.rs

```diff
@@ -662,6 +666,7 @@ impl Document {
         if self.selections.get(&view_id).is_none() {
             self.reset_selection(view_id);
         }
+        self.used_at = std::time::SystemTime::now();
```

Comment:
This seems to only be tracked when the document is first created?

Reply:
no. works on each buffer switch. currently I use this feature right now )
And yes it was not clear. 
Now `doc.used_at` updated only on pick in buffer picker.